coderd/healthcheck/database.go

```diff
@@ -10,20 +10,30 @@ import (
 	"github.com/coder/coder/v2/coderd/database"
 )
 
+const (
+	DatabaseDefaultThreshold = 15 * time.Millisecond
+)
+
 // @typescript-generate DatabaseReport
 type DatabaseReport struct {
-	Healthy   bool    `json:"healthy"`
-	Reachable bool    `json:"reachable"`
-	Latency   string  `json:"latency"`
-	LatencyMs int     `json:"latency_ms"`
-	Error     *string `json:"error"`
+	Healthy     bool    `json:"healthy"`
+	Reachable   bool    `json:"reachable"`
+	Latency     string  `json:"latency"`
+	LatencyMS   int64   `json:"latency_ms"`
+	ThresholdMS int64   `json:"threshold_ms"`
```

Comment:
Should we add the `Threshold string` for consistency? I don't know what is the original though behind string+int64 pairs.

Reply:
Me either, and the fact that I don't know why this particular fence exists makes me pause before knocking it down. I do know that `_ms` was added post-hoc for the UI (8ee500c59d) so I suspect it was simply a case of not renaming a field in the API for compatibility purposes.